core/modules/taxonomy/taxonomy.install

```diff
@@ -400,6 +400,16 @@ function taxonomy_update_1009() {
   }
 }
 
+/**
+ * Clean up extraneous language-related configuration setting.
+ */
+function taxonomy_update_1010() {
+  $config_names = config_get_names_with_prefix('field.bundle.taxonomy_term.');
+  foreach ($config_names as $config_name) {
+    config_clear($config_name, 'extra_fields.display.language');
```

Comment:
I guess, this should have been `extra_fields.form.language`. :wink:

Another thought: there might be settings - wouldn't it be correct to migrate those from language to langcode?
Sure, previously the weight didn't work reliably, but sometimes it worked. Simply throwing the data away doesn't seem right.

Reply:
This suggestion does make sense to me 👍🏼